src/topology/algebra/valuation.lean

```diff
@@ -13,29 +13,26 @@ import ring_theory.valuation.basic
 
 In this file, we define the non archimedean topology induced by a valuation on a ring.
 The main definition is a `valued` type class which equips a ring with a valuation taking
-values in a group with zero (living in the same universe). Other instances are then deduced from
-this.
+values in a group with zero. Other instances are then deduced from this.
 -/
 
 open_locale classical topological_space
 open set valuation
 noncomputable theory
 
-universe u
+universes v u
 
 /-- A valued ring is a ring that comes equipped with a distinguished valuation.-/
-class valued (R : Type u) [ring R] :=
-(Γ₀ : Type u)
-[grp : linear_ordered_comm_group_with_zero Γ₀]
+class valued (R : Type u) [ring R] (Γ₀ : out_param (Type v))
+  [linear_ordered_comm_group_with_zero Γ₀] :=
 (v : valuation R Γ₀)
 
-attribute [instance] valued.grp
-
 namespace valued
-variables {R : Type*} [ring R] [valued R]
+variables {R : Type u} [ring R] (Γ₀ : Type v) [linear_ordered_comm_group_with_zero Γ₀] [valued R Γ₀]
```

Comment:
```suggestion
variables {R : Type u} [ring R] (Γ₀ : Type v) [linear_ordered_comm_group_with_zero Γ₀] [hv : valued R Γ₀]
include hv
```
This seems to work and you can remove `[valued R Γ₀]` from `uniform_space` and `uniform_add_group`.

Reply:
Thanks! It seems that it worked.